tests/foreman/api/test_capsulecontent.py

```diff
@@ -1353,3 +1353,71 @@ def test_positive_remove_capsule_orphans(
             'ls /var/lib/pulp/media/artifact/*/* | xargs file | grep RPM'
         )
         assert result.status, 'RPM artifacts are still present. They should be gone.'
+
+    @pytest.mark.skip_if_not_set('capsule')
+    def test_positive_capsule_sync_openstack_container_repos(
+        self,
+        module_capsule_configured,
+        function_org,
+        function_product,
+        function_lce,
+    ):
+        """Synchronize openstack container repositories to capsule
+
+        :id: 23e64385-7f34-4ab9-bd63-72306e5a4de0
+
+        :setup:
+            1. A blank external capsule that has not been synced yet with immediate download policy.
```

Comment:
```suggestion
            1. A blank external capsule that has not been synced yet.
```
I couldn't find any requirement for `immediate` download policy in the BZ, nor its usage in the test body. I guess we test with the default `on_demand` policy here.

Reply:
Yes, we are not mentioning explicitly which download policy should apply, keeping default